Files/Strings/en-US/Resources.resw

```diff
@@ -1554,6 +1554,9 @@
   <data name="SettingsUseFileListCache.Header" xml:space="preserve">
     <value>Cache files and folders for better performance</value>
   </data>
+  <data name="PreemptiveCacheParallelLimit.Header" xml:space="preserve">
```

Comment:
Why am I not seeing this resource added to all .xlf files?

Reply:
It's actually preferred to let one of us do it when before submitting it to the store, this will prevent potential merge conflicts that are difficult to solve.